release-content/0.15/release-notes/14284_Add_custom_cursors.md

```diff
@@ -1,4 +1,4 @@
 <!-- Add custom cursors -->
 <!-- https://github.com/bevyengine/bevy/pull/14284 -->
 
-<!-- TODO -->
+Previously Bevy's native window cursors supported only a fixed set of built-in OS cursors. Now they also support arbitrary images as "custom cursors". Custom cursors still use native facilities of the OS, which allows them to stay perfectly responsive even when the frame rate of the application drops.
```

Comment:
We should demonstrate how to use this feature, or at the very least link to docs about how to do so.